### PR TITLE
Feature/#272 be refactor studentadditionaltraining endpoint

### PR DESCRIPTION
**Pull request by begoIT24**
We want to apply Model Binding to the GET additional training endpoint

### CODE REVIEW

**Review by ivanlegranbizarro — APPROVED** [review]

File: tests/Feature/Controller/Student/StudentAdditionalTrainingListControllerTest.php [hunk 1]
```diff
@@ -8,60 +8,72 @@
...
+                    'course_beginning_year',
+                    'course_ending_year',
+                    'duration_hrs'
+                ]
+            ]
+        );
     }
 
-    public function testStudentAdditionalTrainingListControllerReturns_404ForInvalidStudentUuid()
+    public function testCanReturn_404WhenStudentIsNotFound()
```
Comment (jordimorillo) [inline tests/Feature/Controller/Student/StudentAdditionalTrainingListControllerTest.php:55]: remove underscores if you are using camelcase

File: app/Http/Resources/AdditionalTrainingCollection.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
+
```
Comment (jordimorillo) [inline app/Http/Resources/AdditionalTrainingCollection.php:2]: declare(strict_types=1);

File: app/Http/Resources/AdditionalTrainingResource.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<?php
+
```
Comment (jordimorillo) [inline app/Http/Resources/AdditionalTrainingResource.php:2]: declare(strict_types=1);

**Review by jordimorillo — CHANGES_REQUESTED** [review]

File: app/Http/Resources/AdditionalTrainingCollection.php (hunk 2 above)
Comment (begoIT24) [reply app/Http/Resources/AdditionalTrainingCollection.php:2]: done

File: app/Http/Resources/AdditionalTrainingResource.php (hunk 3 above)
Comment (begoIT24) [reply app/Http/Resources/AdditionalTrainingResource.php:2]: done

File: tests/Feature/Controller/Student/StudentAdditionalTrainingListControllerTest.php (hunk 1 above)
Comment (begoIT24) [reply tests/Feature/Controller/Student/StudentAdditionalTrainingListControllerTest.php]: done

**Review by jordimorillo — APPROVED** [review]

**Pull request merged**